instructure/canvas-lms
# Allow width/height attributes on <source> tags.

Allow width/height on `<source>` tags so that when contained width a `<picture>` tag the browser can reserve the correct amount of space and not re-layout the page once the image is loaded.

Test plan:

- Create a wiki page and put in <picture> <source> tags. e.g.
```
    <picture>
    <source srcset="https://interactive-examples.mdn.mozilla.net/media/cc0-images/surfer-240-200.jpg" width="240" height="200" media="(max-width: 600px)"/>
    <img src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/painted-hand-298-332.jpg" width="298" height="332" alt="" />
    </picture>
```

- Check that after saving the height/width are persisted in the page.

closes gh-2272
